README.md

```diff
@@ -1,10 +1,12 @@
-# uniffi
+# uniffi - a multi-language bindings generator for rust
```

Comment:
Nice, we can even replace the repo's description too (`A little experiment in more easily building cross-platform components in rust`)

Reply:
Yes, good call, I went ahead and made this change.